map2loop/thickness_calculator.py

```diff
@@ -394,6 +418,8 @@ def __init__(self):
         self.thickness_calculator_label = "StructuralPoint"
```

Comment:
Why are the attributes hard coded? line length should be set per map. Make these variables of the init function with these values as default

Reply:
https://github.com/Loop3D/map2loop/pull/161/commits/d47ac30847e64c69c416528e93646bbe2d95f281

Max line length is now an attribute of the abstract class Default is None for all calculators. structuralPoint needs a value, which is set to 10000m as previous versions.